front/lib/api/assistant/actions/tables_query.ts

```diff
@@ -436,8 +575,64 @@ export class TablesQueryConfigurationServerRunner extends BaseActionConfiguratio
         functionCallName: action.functionCallName,
         agentMessageId: action.agentMessageId,
         step: action.step,
+        resultsFileId: updateParams.resultsFileId
+          ? FileResource.modelIdToSId({
+              id: updateParams.resultsFileId,
+              workspaceId: owner.id,
+            })
+          : null,
+        resultsFileSnippet: updateParams.resultsFileSnippet,
       }),
     };
     return;
   }
 }
+
+async function getTablesQueryOutputCsvFileAndSnippet({
```

Comment:
`nit`
We try to avoid nesting auth in object parameters, but always put it as first argument of the function.

Reply:
fair